### ISSUE TITLE
Trouble connect in adhoc mode in God Eater 2 english patched

### ISSUE THREAD

**Issue by Lemoncak3** [hrydgard/ppsspp#13719]
I download english patched game God eater 2 when try use the adhoc mode the other players don't show in multiplayer lobby. I am using the latest ppsspp build 
![Screenshot_20201127-020610](https://user-images.githubusercontent.com/67026697/100385450-3be2ba00-3055-11eb-96aa-5676d42dd10d.png)

**Comment by Alan-sfs** [comment]
the first issue is that you downloaded a random english patched version, use unpatched ISOs, they mostly work

**Comment by Lemoncak3** [comment]
Hmm the thing is i can't read it if it's in original language so does it will never work then if i use english patched iso?

**Comment by Alan-sfs** [comment]
> Hmm the thing is i can't read it if it's in original language so does it will never work then if i use english patched iso? But toukiden kiwami english patched work though

try another server

**Comment by Lemoncak3** [comment]
Have tried all wlan channel but still can't work

**Comment by MojoJojoDojo** [comment]
If you want good experience over ad-hoc just get a PSP (thats what i did i had a broken one) or use JPCSP on PC because it got a more superior Ad-hoc code and in addition got Infrastructure support too (for future revivels of servers).
I think i gave up on using PPSSPP for now because games keep getting broken from changes.

**Comment by Lemoncak3** [comment]
> If you want good experience over ad-hoc just get a PSP (thats what i did i had a broken one) or use JPCSP on PC because it got a more superior Ad-hoc code and in addition got Infrastructure support too (for future revivels of servers).
> I think i gave up on using PPSSPP for now because games keep getting broken from changes.

I play in android and so far there is a lot of improvement in networking area. There is several titles working fine now and will wait patiently till the team found the solution.

**Comment by MojoJojoDojo** [comment]
![image](https://user-images.githubusercontent.com/33392696/100518545-dd275880-319a-11eb-8ed0-e1256c13c5f3.png)

well locally it does work and I can see the other player.
Maybe try changing the port offset to something else (like 10000).
Or try the networking options first force connect or TCP No Delay sometimes they can help.
I dont know if you are playing over the internet or locally though.

**Comment by Lemoncak3** [comment]
I am using port offset 1100 and usually it worked with other game, and also i have enabled first forced connect and tcp delay. Will try it again later, you are using the english patched iso too @MojoJojoDojo ? I play using local network with my sisters and brothers usually, and games like god eater burst, phantasy star portable, toukiden kiwami, and monster hunter 3 worked just fine.

**Comment by MojoJojoDojo** [comment]
Yeah it works fine with the translated game but I got issues with a low port offset.
It gave me an error.
So maybe it is related.

**Comment by Lemoncak3** [comment]
Okay i'll try it later, thanks for the info. Will inform the result later

**Comment by MojoJojoDojo** [comment]
You can also follow my compatibility list I helped with for multiplayer/ad-hoc games : 
https://docs.google.com/spreadsheets/d/1FTPprUvf551f0aoSpQcmb_qfPP6Ef09ePraXsLbIIx4/edit?usp=sharing
I also got another list but it's not as updated as the above one : 
https://docs.google.com/spreadsheets/d/1PvBsDWtA4VueBtr9vz_aLPIXwpkgjxMCyQOy8sIMaok/edit?usp=sharing

**Comment by ANR2ME** [comment]
> 
> 
> I am using port offset 1100 and usually it worked with other game, and also i have enabled first forced connect and tcp delay. Will try it again later, you are using the english patched iso too @MojoJojoDojo ? I play using local network with my sisters and brothers usually, and games like god eater burst, phantasy star portable, toukiden kiwami, and monster hunter 3 worked just fine.

Did any of the players seeing "Faild to bind port ..." red message on screen? if they're you guys need to change the port offset.
Just because 1100 worked on some games it doesn't mean it will work on other games too.

Port offset is just an addition to shift the original ports, the actual ports being used = `original_port + port_offset`, since different games uses different `original_port` the actual port being used will also be different on different games, thus some of them could bump into an already used port by android or other apps.

**Comment by MojoJojoDojo** [comment]
Yeah the list i linked also says that the same port offset won't work for every game.
I think that Patapon 3 also got that issue.

**Comment by Lemoncak3** [comment]
Sorry got really busy and have the time just now, i tried with 1.12.3 - 230 revision and still not worked even with 10000 port offset. Still stuck at the same screen after players join and not getting into game

**Comment by MojoJojoDojo** [comment]
> Sorry got really busy and have the time just now, i tried with 1.12.3 - 230 revision and still not worked even with 10000 port offset. Still stuck at the same screen after players join and not getting into game

The guy here says it works for him: 
https://github.com/hrydgard/ppsspp/issues/14641#issuecomment-981090981

Don't know whats the issue or maybe its with the english patch.

**Comment by Lumino5501** [comment]
@Lemoncak3 is this issue still not fixed?
